docs/dg/dev/backend-development/data-manipulation/data-publishing/implement-publish-and-synchronization.md

```diff
@@ -289,239 +251,201 @@ class GlossaryWritePublisherPlugin extends AbstractPlugin implements PublisherPl
     public function getSubscribedEvents(): array
     {
         return [
-            GlossaryStorageConfig::ENTITY_SPY_GLOSSARY_TRANSLATION_CREATE,
-            GlossaryStorageConfig::ENTITY_SPY_GLOSSARY_TRANSLATION_UPDATE,
+            StoreStorageConfig::ENTITY_SPY_STORE_UPDATE,
+            StoreStorageConfig::ENTITY_SPY_STORE_CREATE,
```

Comment:
mostly a question

I see in the plugin that there is +1 event `StoreStorageConfig::STORE_PUBLISH_WRITE`, what is it needed? do we have to explain this in the doc as well?